docs/manual-setup.adoc

```diff
@@ -8,7 +8,6 @@ If you are not on macOS or you don't have Homebrew, you need to install:
 - Go compiler, at least 1.14
 - Geth (Ethereum client) 1.9.9-stable
 - Solidity, at least 0.5.17
-- Truffle, at least 5.1.18
```

Comment:
I think we should do the same for the auto deployment and remove truffle from Packer provisioning scripts.